### PR TITLE
[PropertyAccess] Add missing property to PropertyAccessor

### PR DESCRIPTION
**Pull request by vudaltsov**
| Q             | A
| ------------- | ---
| Branch?       | 4.2
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Component/PropertyAccess/PropertyAccessor.php [hunk 1]
```diff
@@ -795,7 +796,7 @@ private function getPropertyPath($propertyPath): PropertyPath
      *
      * @return AdapterInterface
      *
-     * @throws RuntimeException When the Cache Component isn't available
+     * @throws \RuntimeException When the Cache Component isn't available
```
Comment (fabpot) [inline src/Symfony/Component/PropertyAccess/PropertyAccessor.php]: Looks like we are throwing a `LogicException`

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/PropertyAccess/PropertyAccessor.php (hunk 1 above)
Comment (vudaltsov) [reply src/Symfony/Component/PropertyAccess/PropertyAccessor.php]: @fabpot, fixed

**Comment by vudaltsov** [comment]
Status: Needs Review

**Review by Simperfit — APPROVED** [review]
don't we have a test case for the exception ?

**Review by nicolas-grekas — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @vudaltsov.

**Pull request merged**